intro/scipy/index.rst

```diff
@@ -751,88 +751,116 @@ whether the means of two sets of observations are significantly different::
     True
 
-Other integration schemes are available:
-:func:`scipy.integrate.fixed_quad`, :func:`scipy.integrate.quadrature`,
-:func:`scipy.integrate.romberg`...
+Other functions for *numerical quadrature*, including integration of
+multivariate functions and approximating integrals from samples, are available
+in :mod:`scipy.integrate`.
 
-Integrating differential equations
-...................................
+Initial Value Problems
+......................
 
 :mod:`scipy.integrate` also features routines for integrating `Ordinary
 Differential Equations (ODE)
-<https://en.wikipedia.org/wiki/Ordinary_differential_equation>`__. In
-particular, :func:`scipy.integrate.solve_ivp` solves ODE of the form::
+<https://en.wikipedia.org/wiki/Ordinary_differential_equation>`__.
+For example, :func:`scipy.integrate.solve_ivp` integrates ODEs of the form::
 
-    dy/dt = rhs(t, y)
+.. math::
 
-.. note::
+    \frac{dy}{dt} = f(t, y(t))
+
+from an initial time :math:`t_0` and initial state :math:`y(t=t_0)=t_0` to a final
+time :math:`t_f` or until an event occurs (e.g. a specified state is reached).
+
+As an introduction, consider the initial value problem given by
+:math:`\frac{dy}{dt} = -2 y` and the initial condition :math:`y(t=0) = 1` on
+the interval :math:`t = 0 \dots 4`. We begin by defining a callable that
+computes the :math:`f(t, y(t))` given the current time and state.
+
+    >>> def f(t, y):
+    ...     return -2 * y
+
+Then, to compute ``y`` as a function of time::
 
-   For a long time, the go-to method for solving an ODE was
-   :func:`scipy.integrate.odeint`. The SciPy project recommends using
-   :func:`scipy.integrate.solve_ivp` instead.
+    >>> t_span = (0, 4)  # time interval
+    >>> t_eval = np.linspace(*t_span)  # times at which to evaluate `y`
+    >>> y0 = [1,]  # initial state
+    >>> res = sp.integrate.solve_ivp(f, t_span=t_span, y0=y0, t_eval=t_eval)
 
-As an introduction, let us solve the ODE :math:`\frac{dy}{dt} = -2 y` between
-:math:`t = 0 \dots 4`, with the  initial condition :math:`y(t=0) = 1`.
-First the function computing the derivative of the position needs to be defined::
+and plot the result::
 
-    >>> def calc_derivative(time, ypos):
-    ...     return -2 * ypos
+    >>> plt.plot(res.t, res.y[0])  # doctest: +SKIP
+    >>> plt.xlabel('t')  # doctest: +SKIP
+    >>> plt.ylabel('y')  # doctest: +SKIP
+    >>> plt.title('Solution of Initial Value Problem')  # doctest: +SKIP
 
 .. image:: auto_examples/images/sphx_glr_plot_solve_ivp_simple_001.png
     :target: auto_examples/plot_solve_ivp_simple.html
     :scale: 70
     :align: right
 
+Let us integrate a more complex ODE: a `damped
+spring-mass oscillator
+<https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator>`__.
+The position of a mass attached to a spring obeys the 2nd order ODE
+:math:`\ddot{y} + 2 \zeta \omega_0  \dot{y} + \omega_0^2 y = 0` with natural frequency
```

Comment:
Use of $\zeta$ for the [damping ratio](https://en.wikipedia.org/wiki/Damping) is nearly universal; I've not seen $\varepsilon$ used aside from here.